source/includes/_ais.md

```diff
@@ -167,6 +167,39 @@ You'll only be allowed to fetch transactions that were made in the range defined
 `Rejected` transaction status was added in version 3.1.8 of the [Open Banking transactions specification](https://openbankinguk.github.io/read-write-api-site3/v3.1.8/resources-and-data-models/aisp/Transactions.html).
 This status will begin to be returned from the AIS API on the 25th of September 2022.  
 
+The `ProprietaryBankTransactionCode` property has two sub properties `Issuer` and `Code`. `Issuer` will always be set to `Monzo` and all the values for `Code` are listed below:
+
+- `3dsecure`
+- `account_interest`
+- `ach`
```

Comment:
Would a TPP ever encounter this given that US accounts aren't in scope for this API?

Reply:
Ah probably not, I've gone ahead and removed them form the list.